README.md

```diff
@@ -19,3 +19,24 @@ We use the hashing trick and project these features over a space of size `4_096`
 
 The logistic regression is trained in the python notebook attached,
 and used to generate `weight.rs`.
+
+# Comparison with [Whatlang](https://github.com/greyblake/whatlang-rs)
+
+
+The following compares the throughput using the simple benchmark found in this repository and the accuracy using [whatlang-corpora](https://github.com/whatlang/whatlang-corpora) benchmark. Overall, Whichlang is about 50x faster and slightly more accurate than Whatlang. 
+
+### Throughput
+
+|                           | Processing Time (µs) | Throughput (MiB/s) |
+| ------------------------- | -------------------- | ------------------ | 
+| whatlang/short            | 56.229               | 0.48               | 
+| whatlang/long             | 142.1                | 4.1141             | 
+| whichlang/short           | 0.261                | 105.69             | 
+| whichlang/long            | 5.205                | 112.31             | 
+
+### Accuracy
+
+|            | Size <= 20 | 21-50  | 51-100 | Size > 100 | AVG   |
+| -----------| ---------- | ------ | ------ | ---------- | ----- |
```

Comment:
I am uncomfortable with this accuracy data.
It is computed from the dataset we used for training. Of course we do a train/test split... but this is not sufficient at all to do a fair comparison.